### PR TITLE
Pymde

### PR DESCRIPTION
**Pull request by adamgayoso**

### CODE REVIEW

File: docs/api/user.rst [hunk 1]
```diff
@@ -92,6 +92,16 @@ Here we maintain a few package specific utilities for feature selection, etc.
    data.organize_cite_seq_10x
    data.organize_multiome_anndatas
 
+Utilities
+~~~~~~~~~~
+
+Here we maintain a miscellaneous general methods.
```
Comment (jjhong922) [inline docs/api/user.rst:98]:
```suggestion
Here we maintain miscellaneous general methods.
```

or have i been using miscellaneous wrong my whole life

File: scvi/model/utils/_mde.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    This function is included in scvi-tools to provide an alternative to UMAP/TSNE that is GPU-
+    accelerated. The appropriateness of use of visualization of high-dimensional spaces in single-
+    cell omics remains an open research questions. See:
+
+    Chari, Tara, Joeyta Banerjee, and Lior Pachter. "The specious art of single-cell genomics." bioRxiv (2021).
+
+    If you use this function in your research please cite:
+
+    Agrawal, Akshay, Alnur Ali, and Stephen Boyd. "Minimum-distortion embedding." arXiv preprint arXiv:2103.02559 (2021).
+
```
Comment (jjhong922) [inline scvi/model/utils/_mde.py:41]: Add Examples section

**Review by jjhong922 — COMMENTED** [review]
lgtm minor comments

File: docs/api/user.rst (hunk 1 above)
Comment (adamgayoso) [reply docs/api/user.rst:98]: no lol

Comment (adamgayoso) [reply docs/api/user.rst:98]: it's me

**Pull request merged**